vividus-plugin-browserstack/src/main/java/org/vividus/bdd/browserstack/steps/DebugSteps.java

```diff
@@ -62,11 +70,39 @@ public DebugSteps(BrowserStackAutomateClient appAutomateClient, IBddVariableCont
      */
     @When("I save BrowserStack network logs to $scopes variable `$variableName`")
     public void saveNetworkLogs(Set<VariableScope> scopes, String variableName) throws BrowserStackException
+    {
+        getNetworkLogs(logs -> bddVariableContext.putVariable(scopes, variableName, logs));
+    }
+
+    /**
+     * Saves network logs from an application session into JSON context. The application session must be closed before
+     * network logs can be saved.
+     * <br/>
+     * The following steps can be used to work with data in JSON context:
+     * <ul>
+     * <li>Then JSON element by JSON path `$jsonPath` is equal to `$expectedData`$options</li>
+     * <li>Then number of JSON elements by JSON path `$jsonPath` is $comparisonRule $elementsNumber</li>
+     * <li>When I save JSON element from context by JSON path `$jsonPath` to $scopes variable `$variableName`</li>
+     * <li>When I set number of elements found by JSON path `$jsonPath` to $scopes variable `$variableName`</li>
+     * </ul>
+     * @throws BrowserStackException if error occurred while interacting with BrowserStack
+     */
+    @When("I save BrowserStack network logs to JSON context")
+    public void saveNetworkLogsToJsonContext() throws BrowserStackException
+    {
+        getNetworkLogs(logs ->
+        {
+            jsonContext.putJsonContext(logs);
+            attachmentPublisher.publishAttachment(logs.getBytes(StandardCharsets.UTF_8),
+                    "BrowserStack network logs.json");
```

Comment:
is it `har`?

Reply:
in browser stack it's just txt, by its structure it corresponds to HAR format, should I rename it to ....har?